Source/Fuse.Marshal/Marshal.Cast.uno

```diff
@@ -114,6 +114,75 @@ namespace Fuse
 
 			throw new MarshalException(o, typeof(float4));
 		}
+		
+		/**
+			Converts a value to a float4
+			
+			@return true if converted successfully, false if no suitable conversion exists
+			@param value the result value (0 padded as necessary)
+			@param size the size of the result
+		*/
+		public static bool TryMarshalToFloat4(object o, out float4 value, out int size)
```

Comment:
For the `attract` function, where this new code originate, it's a large difference. You need to zeros to animate correctly.

For the math functions it won't make a difference, the result is the same either way. But I prefer this new function since it gives out the size explicitly. The math functions don't change the size of the input then, a float2 remains a float2. In a way this function isn't really producing a `float4` it's just using it for encoding.

However, for value assignment, to a property, the current ToFloat4 makes sense. Since it provides the standard shortcuts of `Margin="4"` and `Margin="4,5"` for example. This function  is actually producing a `float4` for comsumption.